src/structures/Inode.cpp

```diff
@@ -36,3 +40,14 @@ int Inode::get_blocks_amount() {
     return blocks_amount;
 }
 
+void Inode::increase_blocks_amount(){
+    blocks_amount += 1;
+}
```

Comment:
I think it's kinda okay? I mean, in some situations, this abstraction can make code more clear.

Reply:
Well, ok, I do not insist. I just think that +1 is clearer, because at first sight I didn't catch that this function increase blocks_amount only once.